packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -0,0 +1,153 @@
+    .openUrl(`${E2eHelpers.baseUrl}/my-application`)
+    // use Timeouts constants
+    .waitForElementVisible('body', Timeouts.normal)
+    .assert.title('My Application Title | Veterans Affairs')
+    .click('.schemaform-start-button');
+
+  // disable scrolling
+  FormsTestHelpers.overrideFormsScrolling(client);
+
+  // assert navigation is successful
+  E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+  // Personal Information page.
+  client.expect.element('input[name="root_veteranFullName_first"]').to.be
+    .visible;
+  // use functions from the helper file to perform all actions on the page
+  MyApplicationHelpers.completePersonalInformation(client, testData.data);
+  client.axeCheck('.main').click('.form-panel .usa-button-primary');
+  E2eHelpers.expectNavigateAwayFrom(
+    client,
+    '/my-application/personal-information',
+  );
+  ```
+
+**my-application-helpers.js**
+```
+const mock = require('platform/testing/e2e/mock-helpers');
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+const Auth = require('platform/testing/e2e/auth.js');
+const moment = require('moment');
+
+function completePersonalInformation(client, data) {
+  client
+    .waitForElementVisible(
+      'input[name="root_veteranFullName_first"]',
+      Timeouts.normal,
+    )
+    .fill(
+      'input[name="root_veteranFullName_first"]',
+      data.veteranFullName.first,
+    )
+    .fill('input[name="root_veteranFullName_last"]', data.veteranFullName.last)
+}
+```
+_These are recommendations not requirements._
+
+- separate navigation from field input
+  - use a **main test file** for navigation, assertions, and calls helpers
+  - use a **helper file** for filling out forms
+- create separate, numbered **main test files** to organize tests by their focus:
+  - **00-all-fields.e2e.spec.js** - required and optional fields
+  - **01-required.e2e.spec.js** - only required fields
+  - **02-accessibility.e2e.spec.js** - validates accessibility
+  - **03-auth.e2e.spec.js** - validates authentication
+  - **04-cross-cutting-feature.e2e.spec.js** - validates one feature used across several pages (e.g. save in progress)
+- export the end-to-end test using `E2eHelpers.createE2eTest()`- see [Helpers](#helpers)
+- group tests by pages and use a comment to indicate what page is being tested
+- mock all api responses before starting the test. See [Mocking API responses](#mockimg-api-responses)
+- use `waitForElementVisible` before interacting with any element on the page
+- use `Timeouts` constants for setting timeouts (`platform/testing/e2e/timeouts.js`)
+- use helpers for filling data and performing actions on the page
+- perform `axeCheck` on the main body of the application on each page - see [Custom Nightwatch commands](#custom-nightwatch-commands)
+- assert that each navigation is successful
+
+## Mocking API responses
+
+**my-application-helpers.js**
+
+A mock server runs with the end-to-end tests to allow tests to make production-like calls.
```

Comment:
We may want to mention somewhere that you should kill vets-api if you're running it locally, when you want to run an e2e test.